cmd/apps/grafana_app.go

```diff
@@ -30,19 +31,36 @@ func MakeInstallGrafana() *cobra.Command {
 	grafana.Flags().StringP("namespace", "n", "grafana", "The namespace to install grafana")
 	grafana.Flags().Bool("update-repo", true, "Update the helm repo")
 	grafana.Flags().Bool("persistence", false, "Make grafana persistent")
+	grafana.Flags().StringArray("set", []string{},
+		"Use custom flags or override existing flags \n(example --set persistence.enabled=true)")
 
 	grafana.RunE = func(command *cobra.Command, args []string) error {
 
 		const chartVersion = "5.0.4"
 
 		// Get all flags
-		kubeConfigPath, _ := command.Flags().GetString("kubeconfig")
+		kubeConfigPath, err := command.Flags().GetString("kubeconfig")
```

Comment:
Could you move the error handling into `PreRunE` - there are some examples you can follow in the code elsewhere.

Reply:
Done